// discord-images/lib/functions.js
const resolve = require('./resolve.js')

module.exports = {

    baka: function () {
        return resolve.get('baka');
    },
    blush: function () {
        return resolve.get('blush')
    },
    cuddle: function () {
        return resolve.get('cuddle')
    },
    bite: function () {
        return resolve.get('bite')
    },
    dance: function () {
        return resolve.get('dance')
    },
    slap: function() {
        return resolve.get('slap')
    },
    bonk: function() {
        return resolve.get('bonk')
    },
    bully: function() {
        return resolve.get('bully')
    },
    hug: function() {
        return resolve.get('hug')
    },
    kiss: function() {
        return resolve.get('kiss')
    },
    pat: function() {
        return resolve.get('pat')
    },
    wallpaper: function() {
        return resolve.get('wallpaper')
    },
    happy: function() {
        return resolve.get('happy')
    },
    wink: function() {
        return resolve.get('wink')
    },
    kill: function() {
        return resolve.get('kill')
    },
    highfive: function() {
        return resolve.get('highfive')
    },
    wave: function() {
        return resolve.get('wave')
    },
    cry: function() {
        return resolve.get('cry')
    },
    smile: function() {
        return resolve.get('smile')
    },
    punch: function() {
        return resolve.get('punch')
    },
    kick: function() {
        return resolve.get('kick')
    },
    poke: function() {
        return resolve.get('poke')
    },
    goodnight: function() {
        return resolve.get('goodnight')
    },
    zerotwo: function() {
        return resolve.get('zerotwo')
    },
    confused: function() {
        return resolve.get('confused')
    },
    sad: function() {
        return resolve.get('sad')
    },
    thinking: function() {
        return resolve.get('thinking')
    },
    yes: function() {
        return resolve.get('yeah')
    },
    yeet: function() {
        return resolve.get('yeet')
    },
    stare: function() {
        return resolve.get('stare')
    },
    bored: function() {
        return resolve.get('bored')
    },
    scream: function() {
        return resolve.get('scream')
    },
    nervous: function() {
        return resolve.get('nervous')
    },
    angry: function() {
        return resolve.get('angry')
    },
    cringe: function() {
        return resolve.get('cringe')
    },
    glomp: function() {
        return resolve.get('glomp')
    },
    pgif: function() {
        return resolve.get('pgif')
    },
    anal: function() {
        return resolve.get('anal')
    },
    fourk: function() {
        return resolve.get('fourk')
    },
    ass: function() {
        return resolve.get('ass')
    },
    gonewild: function() {
        return resolve.get('gonewild')
    },
    pussy: function() {
        return resolve.get('pussy')
    },
    tigh: function() {
        return resolve.get('tigh')
    },
    hentaiass: function() {
        return resolve.get('hentaiass')
    },
    hentai: function() {
        return resolve.get('hentai')
    },
    hentaitigh: function() {
        return resolve.get('hentaitigh')
    },
    erokemo: function() {
        return resolve.get('erokemo')
    },
    kitsune: function() {
        return resolve.get('kitsune')
    },
    lewd: function() {
        return resolve.get('lewd')
    },
    nekofeet: function() {
        return resolve.get('nekofeet')
    },
    nekopussy: function() {
        return resolve.get('nekopussy')
    },
    nekotits: function() {
        return resolve.get('nekotits')
    },
    solo: function() {
        return resolve.get('solo')
    },
    randomHentaiGif: function() {
        return resolve.get('randomHentaiGif')
    },
    nekoGif: function() {
        return resolve.get('nekoGif')
    },
    lesbian: function() {
        return resolve.get('lesbian')
    },
    cumsluts: function() {
        return resolve.get('cumsluts')
    },
    classic: function() {
        return resolve.get('classic')
    },
    boobs: function() {
        return resolve.get('boobs')
    },
    bj: function() {
        return resolve.get('bj')
    },
    tickle: function() {
        return resolve.get('tickle')
    },
    neko: function() {
        return resolve.get('neko')
    },
    meow: function() {
        return resolve.get('meow')
    },
    lizard: function() {
        return resolve.get('lizard')
    },
    feed: function() {
        return resolve.get('feed')
    },
    why: function() {
        return resolve.get('why')
    },
    fact: function() {
        return resolve.get('fact')
    },
    woof: function() {
        return resolve.get('woof')
    },
    goose: function() {
        return resolve.get('goose')
    },
    avatar: function() {
        return resolve.get('avatar')
    },
    waifu: function() {
        return resolve.get('waifu')
    },
    gecg: function() {
        return resolve.get('gecg')
    },
    holo: function() {
        return resolve.get('holo')
    },
    spoiler: function() {
        return resolve.get('spoiler')
    },
    kemonomimi: function() {
        return resolve.get('kemonomimi')
    },
    catboy: function() {
        return resolve.get('catboy')
    }
}
